packages/shared/src/lib/core/wallet/actions/mintNativeToken.ts

```diff
@@ -13,16 +13,15 @@ import { getTransactionOptions, sendPreparedTransaction } from '../utils'
 import { getActiveNetworkId } from '@core/network'
 
 export async function mintNativeToken(
-    maximumSupply: number,
-    circulatingSupply: number,
+    maximumSupply: string,
```

Comment:
another place mint should just be bigint

Reply:
Relates to mint token details interface.